README.md

```diff
@@ -36,27 +37,61 @@ In particular, you can provide them with `--release`, `--dev`, or `--profile`
 flags to switch the compilation profile. If not option is provided, the scripts
 default to the `--release` profile.
 
-### Running application and examples
 For best experience, it is recommended to use the 
-`scripts/watch.sh --dev` in a second shell. In order to build the IDE application, 
-follow the steps below:
+`scripts/watch.sh --dev` in a second shell.
+
+### Web Application
+In order to build the IDE web application, follow the steps bellow:
 
 ```bash
-cd app
-npm install
-npm run start 
+ide/app$ npm install
+ide/app$ npm run web:dev 
 ```
 
 You can now navigate to http://localhost:8080 and play with it! The example
-scenes will be available at http://localhost:8080/debug.
+scenes will be available at http://localhost:8080/?debug.
```

Comment:
Why this change? It is IMO much uglier than it was. I would like to have a final application in the cloud also following the convention that you can just go to `enso.org/app/debug` to access the debug screen. What are the rationales behind this change?

Reply:
I need to play with it. I will do it on evening.

Reply:
Do we still need it?